scalableminds/webknossos
# Ensure appropriate width of table columns

This PR sets a fixed width to all columns of the exploration and dataset table. Thereby fixes followup of issue  #4106.

### URL of deployed dev instance (used for testing):
- https://maketablesscrollable.webknossos.xyz/

### Steps to test:
- Open the dataset table and resize your window. In fullscreen, the whole table should be visible without a scrolling bar at the bottom of the page. Making the window smaller, the scrollbar should appear and all columns should still have a reasonable width.
- Same goes for the explorative annotations table. Please check both of them :)
- Compared to the current version the difference can clearly be seen in the active and public column of the dataset table.


- Please also check if you think the fixed width of each column is appropriate or should be changed.

### Issues:
- fixes  #4106

------
